cpp/examples/parquet_io/timer.hpp

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Moved back from examples-wide utilities to keep all examples self contained. Ref https://github.com/rapidsai/cudf/pull/20011#issuecomment-3307182911